### PR TITLE
adding protobuf file for receiving RSM-012 message in measurements

### PR DESCRIPTION
**Pull request by andpou**
first take on RSM-012 protobuf file

### CODE REVIEW

**Comment by awha86** [comment]
Updated with new data: 
image

File: source/Contracts/NotifyValidatedMeasureData.proto [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  Timestamp registration = 4;
+  MeteringPointType metering_point_type = 5;
+  Unit quantity_unit = 6;
+  Resolution resolution = 7;
+  TimeInterval timeSeries_time_interval = 8;
+  repeated Points time_series_points = 9;
+
+}
+
+message Points {
```
Comment (HenrikSommer) [inline source/Contracts/NotifyValidatedMeasureData.proto:20]:
```suggestion
  repeated Point time_series_points = 9;

}

message Point {
```

File: source/Contracts/NotifyValidatedMeasureData.proto [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  int64 position = 1;
```
Comment (HenrikSommer) [inline source/Contracts/NotifyValidatedMeasureData.proto:21]: Current `max(position)` for all transactions in _prod_001_ for metering data received from DataHub 2 is 104640, so maybe `int32` is fine, too?

File: source/Contracts/NotifyValidatedMeasureData.proto [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+message NotifyValidatedMeasureData {
+  string orchestration_instance_id = 1;
+  string metering_point_id = 2;
+  string transaction_id = 3;
+  Timestamp registration = 4;
+  MeteringPointType metering_point_type = 5;
+  Unit quantity_unit = 6;
+  Resolution resolution = 7;
+  TimeInterval timeSeries_time_interval = 8;
```
Comment (HenrikSommer) [inline source/Contracts/NotifyValidatedMeasureData.proto:15]:
```suggestion
  TimeInterval time_series_time_interval = 8;
```

File: source/Contracts/NotifyValidatedMeasureData.proto
Comment (HenrikSommer) [inline source/Contracts/NotifyValidatedMeasureData.proto:1]: Et det besluttet at det hedder `MeasureData`?

Comment (andpou) [reply source/Contracts/NotifyValidatedMeasureData.proto:1]: Navnet er taget direkte fra edi guiden

**Review by HenrikSommer — APPROVED** [review]

**Pull request merged**